app/src/main/java/uk/gov/android/authentication/AppAuthSession.kt

```diff
@@ -95,9 +95,8 @@ class AppAuthSession(
             requireNotNull(response.accessTokenExpirationTime) {
                 "Token expiry must not be empty"
             },
-            idToken = requireNotNull(response.idToken) { "id token must not be empty" },
-            refreshToken = response.refreshToken,
-            scope = requireNotNull(response.scope) { "scope must not be empty" }
+            idToken = response.idToken,
```

Comment:
What changed that the `idToken` can be null now?

Reply:
talking with Jamie about the response that comes back. Initally the STS will not return an id_token, this may change in the future though